### ISSUE TITLE
Big Self-Supervised Models are Strong Semi-Supervised Learners 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#88]
# TL;DR
They proposed SimCLRv2, which uses only a small number of labels and performs as well or better than supervised learning. It consists of three stages: unsupervised learning, FineTune, and self-training distillation using unlabeled data. Basically, larger models are better.
![image](https://user-images.githubusercontent.com/24524018/92561123-94a08300-f2ae-11ea-9e34-7c73e98eea8a.png)
![image](https://user-images.githubusercontent.com/24524018/92561135-98cca080-f2ae-11ea-8a3f-f1bea1ac6988.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2006.10029

# Submission Dates(yyyy/mm/dd)

# Authors and institutions

# Methods

# Results

# Comments
